packages/angular/build/src/utils/index-file/auto-csp_spec.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import { autoCsp } from './auto-csp';
+
+describe('auto-csp', () => {
+  it('should rewrite a single inline script', async () => {
+    const result = await autoCsp(`
+      <html>
+        <head>
+        </head>
+        <body>
+          <script>console.log('foo');</script>
+          <div>Some text </div>
+        </body>
+      </html>
+    `);
+
+    expect(result).toContain(
+      `<meta http-equiv="Content-Security-Policy" content="script-src 'strict-dynamic' 'sha256-1kOLrDKT3TBiHLcnxiGsc7HF/lyVJKLhoZDSn0UwCfo=' https: 'unsafe-inline';object-src 'none';base-uri 'self';">`,
```

Comment:
Suggestion: Rather than hard-coding the hashes which don't mean anything to humans anyways, could we replace this with something like:

```typescript
export(result).toContain(`... '${hash('console.log(\'foo\');')}' ...`);
```

I get that this runs the risk that the `hash` function is wrong, but I think that's outweighed by making the tests easier to read and avoid hard-coding or having to update hashes manually.

Reply:
Done!